tests/xfstests/generate_report.pm

```diff
@@ -54,16 +47,7 @@ sub upload_subdirs {
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
-    sleep 5;
-
-    # Reload uploaded status log back to file
-    script_run('curl -O ' . autoinst_url . "/files/status.log; cat status.log > $STATUS_LOG");
-
-    # Reload test logs if check missing
-    script_run("if [ ! -d $LOG_DIR ]; then mkdir -p $LOG_DIR; curl -O " . autoinst_url . '/files/opt_logs.tar.gz; tar zxvfP opt_logs.tar.gz; fi');
```

Comment:
And another issue is, the filename in "Uploaded Logs" should be log-btrfs.tar.xz and test-status.log. But not log and test.

Reply:
It will not! Go trough the VRs and who me where it failed like that ?
It's really disturbing you are linking job which failed due to some api failure, and you don't have the results there with or without this line ...